content/applications/websites/livechat/chatbots.rst

```diff
@@ -222,15 +228,21 @@ Add chatbot to a channel
 
 After a chatbot has been created and tested, it needs to be added to a live chat channel.
 
-First, open the :menuselection:`Live Chat` application, and select the kanban card for a
-:guilabel:`Channel`, or create a :doc:`new one </applications/websites/livechat>`. Click on the
-:guilabel:`Channel Rules` tab. Then, open an existing rule, or create a new one by clicking
-:guilabel:`Add a line`.
+First, open the :menuselection:`Live Chat` application, select the Kanban card for a
```

Comment:
This is a touch confusing because you don't select the card- you hover over it to reveal the 3 dots, then click the dots. Just a suggestion to reword this.